assets/src/edit-story/panels/elements/actionButton.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * External dependencies
+ */
+import styled from 'styled-components';
+
+const ActionButton = styled.button.attrs( { type: 'button' } )`
+	color: ${ ( { theme } ) => theme.colors.mg.v1 };
+	font-size: 11px;
+`;
+
+export default ActionButton;
```

Comment:
I wonder if we should keep all the buttons under `components/button` even if it's only used here at this moment. Thoughts?

Reply:
It's not a "final" button anyway. It's not styled according to Figma, it's just a button for as it was needed. Once someone fixes the design for panel elements and layout, all this will be cleaned up.